components/cluster/libesmtp/libesmtp.p5m

```diff
@@ -13,6 +13,8 @@
 # Copyright 2016 Andreas Grueninger, Grueninger GmbH, (grueni). All rights reserved.
```

Comment:
You probably want to update your copyright.
You also need to set COMPONENT_REVISION